nav2_bringup/params/nav2_params.yaml

```
@@ -96,6 +96,14 @@ bt_navigator:
     - nav2_back_up_cancel_bt_node
     - nav2_drive_on_heading_cancel_bt_node
 
+    navigator: ["navigate_to_pose", "navigate_through_poses"]
```

```suggestion
    navigators: ["navigate_to_pose", "navigate_through_poses"]
```

Update parameter to be plural